bot/exts/pride/prideleader.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot.constants import Colours
+log = logging.getLogger(__name__)
```

Comment:
A bit a spacing helps to distinguish between what is an import statement and logger.
```suggestion
from bot.constants import Colours

log = logging.getLogger(__name__)
```